config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
 
+  get 'errors/not_found'
```

Comment:
Could we keep the routes file organised? Don't add always just to the top. Routes file has to be organised based on priority and matching.